docs/decisions/0001-record-architecture-decisions.rst

```diff
@@ -0,0 +1,31 @@
+Record Architecture Decisions
```

Comment:
We can ask @nasthagiri's opinion, but I don't think this meta-decision is required any more.  I think we either don't need an 001 now, or 001 should describe the decision of why the repo was created to begin with.

Reply:
Makes sense. I saw the suggestion elsewhere as well.  Since we've adopted ADRs throughout the codebase, it's now superfluous to repeat that we're adopting it.